pyproject.toml

```diff
@@ -29,6 +29,7 @@ fastapi-mqtt = ">=1.1"
 # jinja2 = {version = ">=3.0.0", optional = true}
 # fastapi-mqtt = {version = ">=1.0.7", optional = true}
 sqlalchemy = "^2.0.15"
+requests = ">=2.31.0"
```

Comment:
For now it should do the trick, if you are OK with that.
In a near future, it could be interesting to split dependencies based on outputs. For instance, when using the `prom` output, installing FastAPI or even uvicorn doens't make sense. As a user, I know what output(s) I'll use, so it could be a good way of installing only strictly required third-party modules. WDYT?

Reply:
yeah i was think about this - i even started implemented, but didnt really get it done (especially some of the 'heavier' dependencies

Reply:
I could give a try if you are not yet working on it.

Reply:
Im not working on it at the moment